### ISSUE TITLE
Fix IllegalArgumentException in VayraScreeningFormation

### ISSUE THREAD

**Issue by RkShaRkz** [RkShaRkz/Starsector-IntelliJ-Template#39]
The stacktrace in question
```
694171 [Thread-3] ERROR com.fs.starfarer.combat.CombatMain  - java.lang.IllegalArgumentException: Please add code for value 3 in VayraScreeningFormation.java
java.lang.IllegalArgumentException: Please add code for value 3 in VayraScreeningFormation.java
	at data.shipsystems.scripts.VayraScreeningFormation.getStatusData(VayraScreeningFormation.java:113)
	at com.fs.starfarer.combat.systems.OOoO.advance(Unknown Source)
	at com.fs.starfarer.combat.entities.Ship.advance(Unknown Source)
	at com.fs.starfarer.combat.CombatEngine.advanceInner(Unknown Source)
	at com.fs.starfarer.combat.CombatEngine.advance(Unknown Source)
	at com.fs.starfarer.combat.CombatState.traverse(Unknown Source)
	at com.fs.state.AppDriver.begin(Unknown Source)
	at com.fs.starfarer.combat.CombatMain.main(Unknown Source)
	at com.fs.starfarer.StarfarerLauncher.super(Unknown Source)
	at com.fs.starfarer.StarfarerLauncher$1.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by RkShaRkz** [comment]
When deploing Clairvoyant-class Comat Carrier with two longbows wings.

**Comment by RkShaRkz** [comment]
Fix merged-in in commit eb310e9 - closing as resolved until it resurfaces.
